drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/RuleModellerPage.java

```diff
@@ -75,6 +76,11 @@ public void prepareView() {
 
     private void setupRuleModeller() {
         view.setupRuleModellerWidget(ruleModeller());
+        if (plugin() instanceof BRLActionColumnPlugin) {
```

Comment:
Personally, I would add a method in `HasRuleModellerPage` to get the description. I think that's a good idea to avoid the `instanceOf` and follow the open/closed principle. However, I know that maybe my approach is a little bit more prolix. What do you think?